pkg/querier/blocks_store_queryable.go

```diff
@@ -896,6 +908,14 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(ctx context.Context, sp *stor
 					"fetched index bytes", indexBytesFetched,
 					"requested blocks", strings.Join(convertULIDsToString(blockIDs), " "),
 					"queried blocks", strings.Join(convertULIDsToString(myQueriedBlocks), " "))
+				if chunkInfo != nil {
+					for i, s := range mySeries {
+						ls := mimirpb.FromLabelAdaptersToLabels(s.Labels)
+						chunkInfo.StartSeries(ls.Get("series_id"))
+						chunkInfo.FormatStoreGatewayChunkInfo(c.RemoteAddress(), s.Chunks)
```

Comment:
Given we create a separate `ChunkInfoLogger` for each store-gateway, what do you think about doing something similar per ingester? Then we could set the store-gateway or ingester name when creating the `ChunkInfoLogger`, and wouldn't need to pass the store-gateway or ingester name here. 

This could also simplify things for the ingester logging: we could instead do the logging in `SeriesChunksStreamReader.GetChunks`, which knows when we've received the last series and so we wouldn't need to pass `lastOne` to `streamingChunkSeries`.

Reply:
I think in the case of ingesters we can potentially get data from hundreds, so the current method feels more efficient, less log message clutter.